https://issues.redhat.com/browse/ACM-5166 navigation correction and deletion of YAML on 2.7

faaden123 opened the pull request.

faaden123 committed:
navigation correction and deletion of YAML on 2.7

chenz4027 commented on clusters/hosted_control_planes/managing_hosted_aws.adoc:
```diff
@@ -17,12 +17,10 @@ You must configure hosted control planes before you can create hosted control pl
-Select *YAML: On* to view content updates as you enter the information in the console.
-
 If you want to add your cluster to an existing cluster set, you must have the correct permissions on the cluster set to add it. If you do not have `cluster-admin` privileges when you are creating the cluster, you must select a cluster set on which you have `clusterset-admin` permissions. If you do not have the correct permissions on the specified cluster set, the cluster creation fails. Contact y ...
 
```
I think you missed the deletion of "When you review your information and optionally customize it before creating the cluster, you can select YAML: On to view the YAML content in the panel. You can edit the YAML file with your custom settings, if you have any updates."

faaden123 replied:
Thank you. Should be good now.

faaden123 committed:
Update managing_hosted_aws.adoc

chenz4027 reviewed (approved):
/lgtm

swopebe reviewed (approved):
Looks great, thank you!

rokej commented on clusters/hosted_control_planes/managing_hosted_aws.adoc:
```diff
@@ -17,12 +17,10 @@ You must configure hosted control planes before you can create hosted control pl
 == Creating a hosted control plane cluster on AWS with the console
 
-To create a hosted control plane cluster from the {mce-short} console, navigate to *Infrastructure* > *Clusters*. On the _Clusters_ page, click *Create cluster* > *Host inventory* > *Hosted* and complete the steps in the console. 
+To create a hosted control plane cluster from the {mce-short} console, navigate to *Infrastructure* > *Clusters*. On the _Clusters_ page, click *Create cluster* > *Amazon Web Services* > *Hosted* and complete the steps in the console. 
```
The whole documentation after this line is not applicable to a hosted control plane cluster on AWS because there is no UI wizard to create a hosted control plane for AWS. The whole documentation after this line is only applicable to BareMetal Agent platform.

rokej replied:
So we should move the original content of `Creating a hosted control plane cluster on AWS with the console` into `managing_hosted_bm.adoc` right under the pre-req section and name the section `Creating a hosted control plane cluster on Bare Metal Agent with the console`. In the doc, there is also a section called `Creating a hosted cluster on bare metal`. We should name that section `Creating a hosted cluster on bare metal with CLI`.

rokej replied:
For BareMetal with Agent platform, we support both UI console and CLI for creating a hosted cluster.

faaden123 replied:
the steps indicated From the previous section" Important" and on detail the usage of the console with the UI providing more than just information. In this case, the UI only provides information on how to create a cluster.  but it does not walk you through it, which is the reason why these steps have been removed. Additionally, the console on bare metal does which is why this information needs to be moved to another document, which Roke has mentioned. This may need to be a separate request. I will tag @swopebe to confirm if this issue needs to be a separate one.

In terms of the note, @swopebe , I have added a mock note to indicate what I stated and what Roke has emphasized in our meeting about making sure the user knows this. I am unsure of a) the usage of the acronym UI, and my language usage in this sentence( It may need to be longer to make sure users understand this). I will look over the guide and come back with some edits, if any. I would appreciate your constructive criticism on this. 

@chenz4027, if you could go ahead and review the changes to this document as you see fit. Thank you all for your help on one of my very first issues here as an Intern:).  I look forward to your responses.

faaden123 committed:
Update managing_hosted_aws.adoc

The steps indicated From the previous section" Important" and on detail the usage of the console with the UI providing more than just information. In this case, the UI only provides information on how to create a cluster.  but it does not walk you through it, which is the reason why these steps have been removed. Additionally, the console on bare metal does which is why this information needs to be moved to another document, which Roke has mentioned. This may need to be a separate request. I will tag @swopebe  to confirm if this issue needs to be a seperate one.
Additionally(@swopebe ) Roke has mentioned adding a note in which states the information presented above. I have added a mock note, and will refer back to the style guide as I am not quite sure of both a) the reference of UI and b)the necessary adjustments to my mock note. Please review but I will also review this and suggest changes.

 @chenz4027 if you could go ahead and review the changes to this document as you see fit. Thank you all for your help on one of my very first issues here as an Intern. I look forward  to your responses.

chenz4027 reviewed (approved):
/lgtm

chenz4027 reviewed (approved):
/lgtm

swopebe commented:
@rokej with your changes, coming in pretty late in the release schedule, this will take more time. I will review this with @faaden123 next week after the US holiday.

faaden123 committed:
Update managing_hosted_bm.adoc

rokej commented on clusters/hosted_control_planes/managing_hosted_aws.adoc:
```diff
@@ -17,31 +17,9 @@ You must configure hosted control planes before you can create hosted control pl
+*Note:* After reaching the console, the UI will only provide information on how to create a cluster.
 
 [#hosted-deploy-cluster-aws]
 == Deploying a hosted cluster on AWS
```
Can we change this to `Deploying a hosted cluster on AWS with CLI`?

rokej commented on clusters/hosted_control_planes/managing_hosted_bm.adoc:
```diff
@@ -20,8 +20,34 @@ You can use the {mce} console to create and manage a {ocp} hosted cluster. Hoste
+[#create-hosted-aws]
+== Creating a hosted control plane cluster on Bare Metal Agent with the console
+
+To create a hosted control plane cluster from the {mce-short} console, navigate to *Infrastructure* > *Clusters*. On the _Clusters_ page, click *Create cluster* > *Amazon Web Services* > *Hosted* and complete the steps in the console. 
```
It should be `Create cluster > Host inventory > Hosted control plane`

rokej commented on clusters/hosted_control_planes/managing_hosted_aws.adoc:
```diff
@@ -17,31 +17,9 @@ You must configure hosted control planes before you can create hosted control pl
-When you review your information and optionally customize it before creating the cluster, you can select *YAML: On* to view the YAML content in the panel. You can edit the YAML file with your custom settings, if you have any updates.  
-
-*Note:* You have to run the `oc` command that is provided with the cluster details to import the cluster. When you create the cluster, it is not automatically configured with the management of {product-title-short}.
+*Note:* After reaching the console, the UI will only provide information on how to create a cluster.
```
After reaching the console, the UI will only provide basic information on how to create a cluster with CLI. Currently there is no UI wizard to create a cluster.

swopebe replied:
To avoid dangling modifier:

After you reach the console, use the basic information that is provided to create a cluster in the command line.


CLI is also something that we should not be using

faaden123 committed:
Update managing_hosted_aws.adoc

faaden123 committed:
Update managing_hosted_bm.adoc

faaden123 committed:
Update managing_hosted_aws.adoc

faaden123 committed:
Update managing_hosted_bm.adoc

faaden123 committed:
Update managing_hosted_aws.adoc
